src/server/serializer_test.ts

```diff
@@ -58,6 +60,60 @@ Deno.test("serializer - signals", async (t) => {
   assertEquals(deserialized.b.peek(), 2);
 });
 
+Deno.test("serializer - @preact/signals-core 1.2.3", async (t) => {
+  const data = {
+    a: 1,
+    b: signal123(2),
+  };
+  const res = serialize(data);
+  assert(res.requiresDeserializer);
+  assert(res.hasSignals);
+  await assertSnapshot(t, res.serialized);
+  const deserialized: any = deserialize(res.serialized, signal);
+  assertEquals(typeof deserialized, "object");
+  assertEquals(deserialized.a, 1);
+  assertEquals(deserialized.b.value, 2);
+  assertEquals(deserialized.b.peek(), 2);
+});
+
+Deno.test("serializer - @preact/signals-core 1.3.0", async (t) => {
+  const data = {
+    a: 1,
+    b: signal130(2),
+  };
+  const res = serialize(data);
+  assert(res.requiresDeserializer);
+  assert(res.hasSignals);
+  await assertSnapshot(t, res.serialized);
+  const deserialized: any = deserialize(res.serialized, signal);
+  assertEquals(typeof deserialized, "object");
+  assertEquals(deserialized.a, 1);
+  assertEquals(deserialized.b.value, 2);
+  assertEquals(deserialized.b.peek(), 2);
+});
+
+Deno.test("serializer - multiple versions of @preact/signals-core", async (t) => {
+  const data = {
+    a: 1,
+    b: signal(2),
```

Comment:
Right now this aliases to `1.3.0` too, but I left it here in case we update the version in the future. That way we'll automatically cover that together with `1.2.3` and `1.3.0` too.